docs/standards/smart-contracts/lsp1-universal-receiver-delegate-up.md

```diff
@@ -33,12 +33,10 @@ function supportsInterface(bytes4 interfaceId) public view returns (bool)
 
 ## Functions
 
-### universalReceiverDelegate
+### universalReceiver
```

Comment:
Since we mention that:

> The **LSP1UniversalReceiverDelegateUP** is a contract called by the **[`universalReceiver(...)`](./lsp0-erc725-account.md#universalreceiver)** function of the **[LSP0ERC725Account](./lsp0-erc725-account.md)** contract.

I think it is worth mentioning that this universalReceiver function will receive extra calldata if called by the universalReceiver function of the LSP0.

We can have the table of parameters, and say if called by LSP0's universalReceiver function:

#### Extra calldata 

| Name     | Type    | Description                                                                  |
| :------- | :------ | :--------------------------------------------------------------------------- |
| `caller` | bytes20 | The token's or vault's smart contract address.                               |
| `value`  | bytes32 | The amount of value sent to the universalReceiver function.                  |

Something like this, let me know your opinion